packages/compass-home/src/contexts/app-registry-context.ts

```diff
@@ -1,9 +1,9 @@
 import { createContext, useContext, useState } from 'react';
 import type React from 'react';
 import AppRegistry from 'hadron-app-registry';
-import debugModule from 'debug';
```

Comment:
You will probably need to remove it from package deps now, doesn't look like it's used anymore in this plugin